ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -446,6 +458,32 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlGetToHtmlDocumentWithSession(SteamCommunityURL, request).ConfigureAwait(false);
 		}
 
+		internal async Task<HashSet<ulong>> GetDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
+			if (response == null) {
+				return null;
+			}
+
+			HtmlNodeCollection nodes = response.DocumentNode.SelectNodes("//div[@class='pending_gift']/div[starts-with(@id,'pending_gift_')]/div[@class='pending_giftcard_leftcol']/../@id");
+			HashSet<ulong> results = new HashSet<ulong>();
+			foreach (string strGID in nodes.Select(node => node.Attributes["id"].Value.Substring(13))) {
```

Comment:
Despite of xpath guarantee, you should still use GetAttributeValue(), it handles edge cases for you. See other functions.